pages/Documents/MessagingChannels/MobileAppMessagingSDKforAndroid/ReleaseNotes/5.7.1.md

```diff
@@ -27,16 +27,17 @@ The Android Mobile Messaging SDK version 5.7.1 uses:
 
 **(unchanged from version 5.7.0)**
 
-# Bugs Fixed:
+# Bugs fixed
+
 - Link previews for consumer messages are not hidden even when the feature is disabled.
 
-# Enhancements:
+# Enhancements
+
 - [Proactive to InApp messaging](mobile-app-messaging-sdk-for-android-advanced-features-proactive-and-ivr-deflection-to-app-messaging.html) feature now has extended to support Rich Content Push Notification messages.
 
-# Known Issue:
+# Known issue
 
-Deep links shared via Structured Content for InApp navigation are failing due to `http` prefix is getting added when clicked on a link.
-The workaround for this issue is to use [structured_content_link_as_callback](mobile-app-messaging-sdk-for-android-sdk-attributes-5-0-and-above.html#structured-content) configuration of SDK and handle links inside host app when fired SDK event:  [LP_ON_STRUCTURED_CONTENT_LINK_CLICKED](mobile-app-messaging-sdk-for-android-sdk-apis-callbacks-index.html#structured-content-link-clicked)
+Deep links shared via Structured Content for InApp navigation are failing due to `http` prefix is getting added when clicked on a link. The workaround for this issue is to use [structured_content_link_as_callback](mobile-app-messaging-sdk-for-android-sdk-attributes-5-0-and-above.html#structured-content) configuration of SDK and handle links inside host app when fired SDK event:  [LP_ON_STRUCTURED_CONTENT_LINK_CLICKED](mobile-app-messaging-sdk-for-android-sdk-apis-callbacks-index.html#structured-content-link-clicked)
```

Comment:
Deep links shared via Structured Content for InApp navigation are failing **because the** `http` prefix is getting added when **clicking** on a link. The workaround for this issue is to use [structured_content_link_as_callback](mobile-app-messaging-sdk-for-android-sdk-attributes-5-0-and-above.html#structured-content) configuration of SDK and handle links inside **the** host app when **firing** **an** SDK event:  [LP_ON_STRUCTURED_CONTENT_LINK_CLICKED](mobile-app-messaging-sdk-for-android-sdk-apis-callbacks-index.html#structured-content-link-clicked)

Some notes above on tenses here - I hope I kept the meaning right. Feel free to change accordingly but please review the tenses and grammar as they do not currently add up. @j9t you cannot say things like "Is getting added when (past tense) clicked on a link"

Reply:
@lkisoslive, note that there was just a line break change here, contents were not changed as part of this PR. In this case we’ll keep it at that. Can you take note so that we fix the problem you see later?

Reply:
Sure. Keeping track on [DX-93](https://jira.liveperson.com/browse/DX-93).